doc/glossary/index.rst

```diff
@@ -31,3 +31,374 @@ Glossary
+
+        Starting with version 3.2 privacyIDEA also provides a :ref:`logger_audit` and
+        a :ref:`container_audit` which can be used to send privacyIDEA audit log messages
+        to services like splunk or logstash.
+
+   Tokenowner
+        The owner of a token is the user for whom the token was rolled out.
+
+   FailCount
+   MaxFail
+        The FailCount count the number of failed login attempts.
+
+        If the login fail counter reaches the ``MaxFail`` the user can not login
+        with this token anymore.
+
+        The administrator or help desk user can select those tokens and
+        click the button ``reset failcounter`` to reset the fail counter to zero.
+        The tokens can be used for authentication again.
+
+   SplitAtSign
+        ``splitAtSign`` defines if the username like *user@company*
+        given during authentication should
+        be split into the loginname *user* and the realm name *company*.
+        In most cases this is the wanted behaviour so this is enabled by default.
+
+        But given your users log in with email addresses like *user@gmail.com* and
+        *otheruser@outlook.com* you probably do not want to split.
+
+        How a user is related to a realm is described here: :ref:`relate_realm`
+
+        This option also affects the login via the :ref:`rest_auth`
+
+   Rollout State
+        A token can be rolled out in several steps like the 2step HOTP/TOTP token.
+        In this case the attribute ``“rollout_state”`` of the token contains certain values
+        like ``‘clientwait’ or ‘enrolled’``. This way actions can be triggered, depending
+        on the step during an enrollment process.
+
+   Custom User Attributes
+        The table ``“customuserattribute”`` is used to store additional, custom attributes for users.
+
+        A user is identified by the user_id, the resolver_id and the realm_id.
+        The additional attributes are stored in Key and Value.
+        The Type can hold extra information like e.g. an encrypted value / password.
+
+        .. note:: Since the users are external, i.e. no objects in this database, there is not
+            logic reference on a database level. Since users could be deleted from user stores without
+            privacyIDEA realizing that, this table could pile up with remnants of attributes.
+
+   Scope
+        A scope is the area, where a policy is meant for.
+        This can be values like:
+
+        * ADMIN = 'admin'
+
+        * AUDIT = 'audit'
+
+        * AUTH = 'authentication'
+
+        * AUTHZ = 'authorization'
+
+        * ENROLL = 'enrollment'
+
+        * REGISTER = 'register'
+
+        * USER = 'user'
+
+        * WEBUI = 'webui'
+
+        scope takes only one value.
+
+   Realms
+        Users need to be in realms to have tokens assigned. A user, who is not
+        member of a realm can not have a token assigned and can not authenticate.
```

Comment:
realms are ment for general logical user grouping.

Reply:
done in 3654d0695eb1bf3fee0a79024198bf292d2d29eb.